charts/kubeai/charts/openwebui/templates/securityContextConstraints.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
+{{- if .Values.serviceAccount.create }}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: openwebui
```

Comment:
Let's template this with fullname like other resources.

Reply:
Done